scripts/appveyor-tool.ps1

```diff
@@ -93,7 +93,23 @@ Function Bootstrap {
   echo "Rtools is now available on drive $RtoolsDrive"
 
   Progress "Setting PATH"
-  $env:PATH = $RtoolsDrive + '\Rtools\bin;' + $RtoolsDrive + '\Rtools\MinGW\bin;' + $RtoolsDrive + '\Rtools\gcc-4.6.3\bin;' + $env:PATH
+  if ( -not(Test-Path Env:\GCC_PATH) ) {
+    echo "Detecting whether C++11 is required"
+    $ErrorActionPreference = "Continue"
+    Rscript -e "stopifnot(grepl('[cC]\\+\\+11', read.dcf('DESCRIPTION')[,'SystemRequirements']))"
+    if ($LastExitCode -ne 0) {
+      $gcc_path = "gcc-4.6.3"
```

Comment:
Sorry it took me so long.

I really think we should keep things simple and stupid here, and use this alternative if `GCC_PATH` is not specified; at some later point, the default might become "mingw_32".